aws/data_source_aws_iam_role.go

```diff
@@ -97,6 +99,10 @@ func dataSourceAwsIAMRoleRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("unique_id", output.Role.RoleId)
 
+	if len(output.Role.Tags) > 0 {
```

Comment:
@robh007 Thanks for this.
`if len(output.Role.Tags) > 0` shouldn't be necessary here as `keyvaluetags.IamKeyValueTags` can handle that case.